nixos/modules/services/continuous-integration/buildkite-agent.nix

```diff
@@ -31,6 +31,14 @@ in
         '';
       };
 
+      hooks-path = mkOption {
+        type = types.str;
+        default = "${pkgs.buildkite-agent}/share/hooks";
```

Comment:
This needs also `defaultText = "${pkgs.buildkite-agent}/share/hooks";` otherwise manual will depend on buildkite-agent package

Reply:
Done!